### ISSUE TITLE
Add option to compute root_mean_squared_error

### ISSUE THREAD

**Issue by johannespitz** [PyTorchLightning/pytorch-lightning#7547]
## 🚀 Feature
Allow the user to choose between MSE and RMSE.

### Motivation
In a physical domain the RMSE, which is essentially the mean of distances, may be significantly more intuitive than the MSE. Therefore, it would be nice to have the option to choose the preferd metric.

### Pitch
Similar to the implementation in [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html#sklearn.metrics.mean_squared_error) one could simply pass `squared=False` to the `MeanSquaredError` module or the `mean_squared_error` function.
